### ISSUE TITLE
CMS - Lighthouse: make CMS the source of truth for facility URLs

### ISSUE THREAD

**Issue by swirtSJW** [department-of-veterans-affairs/va.gov-cms#7279]
## User Story or Problem Statement

As a lighthous facility data engineer, I need the urls of services pages so that consumers of the facility API know where to send people.

This is mostly covered by #8565     The remainder is just to account for Lovell and Manilla if needed.

## Acceptance Criteria
- [x] CMS would carry the exemption in code for  Lovell and Manilla.  So that CMS is always the source of truth.  FOrking should be based on facility id, not name string match.
  - [x] If the system is Lovell it should use https://www.lovell.fhcc.va.gov/  as system url. 
      - [x]   If the locator id = vha_556 (Capt James A Lovell) it should probably use https://www.lovell.fhcc.va.gov as well  
     - [x] vha_556GA (Evanston VA clinic) should use https://www.lovell.fhcc.va.gov/locations/Evanston_Community_Based_Outpatient_Clinic.asp
     - [x] vha_556GD (Kenosha VA Clinic) should use https://www.lovell.fhcc.va.gov/locations/Kenosha_Community_Based_Outpatient_Clinic.asp
     - [x] vha_556GC (McHenry VA Clinic) should use https://www.lovell.fhcc.va.gov/locations/McHenry_Community_Based_Outpatient_Clinic.asp
  - [ ]  If it is Manilla it should push ___  ~Manilla is not a consideration right now as there is no manilla facility in CMS for pushing.~  <--- THis was wrong because I am an idiot and think Manila it spelled with two L like Vanilla

## Implementation steps
<optional>

## Design principles

Veteran-centered
- [ ] `Single source of truth`: Increase reliability and consistency of content on VA.gov by providing a single source of truth.
- [ ] `Accessible, plain language`: Provide guardrails and guidelines to ensure content quality.
- [ ] `Purposely structured content`: Ensure Content API can deliver content whose meaning matches its structure.
- [ ] `Content lifecycle governance`: Produce tools, processes and policies to maintain content quality throughout its lifecycle.

Editor-centered
- [ ] `Purpose-driven`: Create an opportunity to involve the editor community in VA’s mission and content strategy goals.
- [ ] `Efficient`: Remove distractions and create clear, straightforward paths to get the job done.
- [ ] `Approachable`: Offer friendly guidance over authoritative instruction.
- [ ] `Consistent`: Reduce user’s mental load by allowing them to fall back on pattern recognition to complete tasks.
- [ ] `Empowering`: Provide clear information to help editors make decisions about their work.

### CMS Team

Please leave only the team that will do this work selected. If you're not sure, it's fine to leave both selected.

- [ ] `Platform CMS Team`
- [x] `Sitewide CMS Team`

**Comment by kevwalsh** [comment]
Blocked by lighthouse still.

**Comment by kevwalsh** [comment]
We may need to discuss what to do with unpublished Domicilary and Mobile clinic facility URLs.

**Comment by kevwalsh** [comment]
Closing this because the work is done, but PR open as discussed. We can demo this when Jay is back and the PR is merged.

**Comment by dsinla** [comment]
Basically done but blocked by Lighthouse v1

**Comment by swirtSJW** [comment]
On the lighthouse call today, Adam said they are ready to start receiving this data.

**Comment by kmariepat-cityfriends** [comment]
Update from Lighthouse team:
- v0 impact issues delaying URL testing, should be able to start this week of 5/8

**Comment by swirtSJW** [comment]
Testing thread in slack  https://dsva.slack.com/archives/C02BTJTDFTN/p1684274199615489

TLDR it works with Lighthouse sandbox.
